contrib/debian/bitcoin-qt.desktop

```diff
@@ -11,3 +12,4 @@ Type=Application
 Icon=bitcoin128
 MimeType=x-scheme-handler/bitcoingold;
 Categories=Office;Finance;
+StartupWMClass=Bitcoin-qt
```

Comment:
This could be problematic if both bitcoin wallet and bitcoingold wallet are installed and running on the same system.